packages/qwik/src/prefetch-service-worker/direct-fetch.ts

```diff
@@ -79,8 +79,8 @@ function taskTick(swState: SWState) {
     if (task.$isFetching$) {
       outstandingRequests++;
     } else if (
-      outstandingRequests < swState.$maxPrefetchRequests$ ||
-      task.$priority$ >= DIRECT_PRIORITY
+      swState.$getCache$() &&
```

Comment:
This prevents $put$ without cache in place

Reply:
Shouldn't you await that?

Reply:
Yes, nice catch